provisioner/powershell/provisioner.go

```diff
@@ -116,15 +116,15 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 	}
 
 	if p.config.ElevatedEnvVarFormat == "" {
-		p.config.ElevatedEnvVarFormat = `$env:%s="%s"; `
+		p.config.ElevatedEnvVarFormat = `$env:%s=\"%s\"; `
 	}
 
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = `powershell "& { {{.Vars}}{{.Path}}; exit $LastExitCode}"`
+		p.config.ExecuteCommand = `powershell '& {$ProgressPreference=\"SilentlyContinue\"; {{.Vars}}{{.Path}}; exit $LastExitCode}'`
```

Comment:
An update as follows:
I updated my .json file to include the execute_command which solves the problem, now I would just like to understand how and why this works. As its not required with Virtualbox with the standard packer.exe from packer.io.

    {
        "type": "powershell",
        "execute_command": "powershell \"& { {{.Vars}}{{.Path}}; exit $LastExitCode}\"",
		"script": "./scripts/Wait-ForPatching.ps1",
        "pause_before": "60s",
        "environment_vars": [
            "localadminpw={{ user `admin_pass` }}"
        ]
    },